AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/AnyModelIntegrationTests.swift

```diff
@@ -20,7 +20,7 @@ class AnyModelIntegrationTests: XCTestCase {
         Amplify.reset()
         Amplify.Logging.logLevel = .verbose
 
-        let plugin = AWSAPIPlugin(modelRegistration: PostCommentModelRegistration())
+        let plugin = AWSAPIPlugin(schema: PostCommentSchema())
```

Comment:
do we need to rebase changes? in `AWSAPIPlugin.swift`, i'm still seeing:
```
 public init(modelRegistration: AmplifyModelRegistration? = nil,
                sessionFactory: URLSessionBehaviorFactory? = nil)
```

Reply:
I believe we are sticking with modelRegistration for now. reverting this, it looks like it got pulled in due to cherry-picking my commit that was built on the `schema` change that was reverted